common/app/model/content.scala

```diff
@@ -687,17 +694,11 @@ object Gallery {
     val contentOverrides = content.copy(
       metadata = metadata,
       trail = trail,
-      rawOpenGraphImage = {
-        val bestOpenGraphImage = if (FacebookShareUseTrailPicFirstSwitch.isSwitchedOn) {
-          trail.trailPicture.flatMap(_.largestImageUrl)
-        } else {
-          None
-        }
```

Comment:
I am ok with that. I actually encountered this pattern yesterday. 😺 
Not sure everybody would be ok with the syntax though, especially if the conversion is implicit.